pkg/runplugin/runplugin.go

```diff
@@ -202,6 +202,18 @@ func createDeploymentForConfig(config *pluginConfig) *appsv1.Deployment {
 									},
 								},
 							},
+							// NOTE This will provide WAGGLE_NODE_ID and WAGGLE_NODE_VSN for cases that a plugin
```

Comment:
This was accidentally added... but it's in runplugin, which we will eventually drop in place of pluginctl.